modules/core/02-client/client/cli/tx.go

```diff
@@ -241,6 +245,62 @@ func NewUpgradeClientCmd() *cobra.Command {
 	return cmd
 }
 
+// NewCmdRecoverClientProposal defines the command to recover an IBC light client
+func NewCmdRecoverClientProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "recover [subject-client-id] [substitute-client-id] [flags]",
+		Args:  cobra.ExactArgs(2),
+		Short: "recover an IBC client",
+		Long: "Submit a recover IBC client proposal along with an initial deposit.\n" +
+			"Please specify a subject client identifier you want to recover..\n" +
+			"Please specify the substitute client the subject client will be recovered to.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			proposal, err := govcli.ReadGovPropFlags(clientCtx, cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			subjectClientID := args[0]
+			substituteClientID := args[1]
```

Comment:
🤔

feel free to ignore, but I think this is perfect scenario where double assignment in one expression looks better!  

```suggestion
			subjectClientID, substituteClientID := args[0], args[1]
```

Reply:
slam it in